src/onegov/org/forms/directory.py

```diff
@@ -341,6 +350,14 @@ def validate_thumbnail(self, field):
                 _("Please select at most one thumbnail field")
             )
 
+    def validate_numbers(self, field):
```

Comment:
This method seems not to be used anywhere. Probably created for numbering RadioField above right?

Reply:
It gets called upon validation. Just as the method above "validate_thumbnail".